README.rst

```diff
@@ -1,21 +1,27 @@
 ==========================
-Django eduNEXT Audit Model
+Django edunext audit model
 ==========================
 
-.. image:: https://github.com/eduNEXT/eox-audit-model/actions/workflows/tests.yml/badge.svg
-   :target: https://github.com/eduNEXT/eox-audit-model/actions/workflows/tests.yml
+|Maintainance Badge| |Test Badge| |PyPI Badge|
 
+.. |Maintainance Badge| image:: https://img.shields.io/badge/Status-Maintained-brightgreen
+   :alt: Maintainance Status
+.. |Test Badge| image:: https://img.shields.io/github/actions/workflow/status/edunext/eox-audit-model/.github%2Fworkflows%2Ftests.yml?label=Test
+   :alt: GitHub Actions Workflow Test Status
+.. |PyPI Badge| image:: https://img.shields.io/pypi/v/eox-audit-model?label=PyPI
+   :alt: PyPI - Version
+   
+Eox audit model is a Django application that provides an audit model for tracking and logging changes within the Open edX platform.
+This plugin is designed to help administrators and developers maintain a comprehensive audit trail of various operations, ensuring better monitoring.
 
-.. image:: https://github.com/eduNEXT/eox-audit-model/actions/workflows/bump_version.yml/badge.svg
-    :target: https://github.com/eduNEXT/eox-audit-model/actions/workflows/bump_version.yml
+Features
+========
 
-.. image:: https://github.com/eduNEXT/eox-audit-model/actions/workflows/python-publish.yml/badge.svg
-    :target: https://github.com/eduNEXT/eox-audit-model/actions/workflows/python-publish.yml
-    
-.. image:: https://img.shields.io/badge/Status-Maintained-brightgreen
+- Tracks changes to specified models.
+- Logs detailed information about create, update, and delete operations.
 
 Installation
```

Comment:
I think we should add information in this section about how to install this in Tutor, similar to the explanation here: [eox-tenant installation](https://github.com/eduNEXT/eox-tenant/blob/a29fb7c88f469c98e78cab3ead9fbf18da8babb7/README.rst#installation). What do you think?

Reply:
Sorry for the delay, I forgot to answer. 

I have a question about it because the audit model requires direct modification in the code (add the decorator or the method), should be recommended to be installed by tutor `OPENEDX_EXTRA_PIP_REQUIREMENTS`, or should be installed through the plugin or code that wants to use it?

Reply:
I agree with @magajh. I think it would be valuable to briefly explain how to install it with Tutor.

About your question @dcoa, I think that the way of installation we should recommend is with `OPENEDX_EXTRA_PIP_REQUIREMENTS`, so when you have that plugin installed, you can use it in your code as you prefer. Also, we already install audit-model using that config, e.g., [audit-model in quince](https://github.com/eduNEXT/ednx-strains/blob/master/quince/dedalo/config.yml#L132).

Reply:
@dcoa, please ping me when you've addressed this comment so I can leave my approval :)